src/java/org/apache/cassandra/repair/RepairRunnable.java

```diff
@@ -415,10 +414,10 @@ private void normalRepair(UUID parentSession,
             @SuppressWarnings("unchecked")
             public ListenableFuture apply(List<RepairSessionResult> results)
             {
+                logger.debug("Repair result: {}", results);
                 // filter out null(=failed) results and get successful ranges
                 for (RepairSessionResult sessionResult : results)
                 {
-                    logger.debug("Repair result: {}", results);
```

Comment:
Question: Having this here meant we simply didn't log anything if the results were empty? (I wonder if the original intent was to log the individual session results...)

Reply:
>  if the original intent was to log the individual session results...

Might be. I had the individual result logging within the for loop earlier and moved it outside for 2 reasons. 
- log all results in a single message. 
- log the message even if the list is empty so we know that execution step.